server/src/config/tokens.ts

```diff
@@ -0,0 +1,3 @@
+export const TOKENS = {
+  "IPINFO": "fake_token"
```

Comment:
is there a better place to keep this?

Reply:
I'm assuming this is a security token? If so then it should be stored in an environment variable.